src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java

```diff
@@ -111,12 +111,12 @@
     private CcdNotificationWrapper ccdNotificationWrapper;
     private SscsCaseDataWrapper sscsCaseDataWrapper;
     private final Subscription subscription = Subscription.builder()
-        .tya(APPEAL_NUMBER)
-        .email(EMAIL)
-        .mobile(MOBILE_NUMBER_1)
-        .subscribeEmail(YES)
-        .subscribeSms(YES).wantSmsNotifications(YES)
-        .build();
+            .tya(APPEAL_NUMBER)
```

Comment:
There is a lot of formatting changes in this class. Are you using a different intelliJ settings from the rest of the team?

Reply:
Same as everyone

Reply:
I don't know, kind of feel these formatting changes should go in their own PRs in future as makes it very difficult to review these PRs when there are so many lines that have 'changed'